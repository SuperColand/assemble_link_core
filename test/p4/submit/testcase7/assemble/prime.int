.data
0

.text
addu $s0 $0 $a0
addiu $t0 $0 1
beq $s0 $t0 is_not_prime
addiu $t0 $0 2
beq $t0 $s0 is_prime
divu $s0 $t0
mfhi $s1
beq $s1 $zero is_not_prime
addiu $t0 $t0 1
j loop
addu $t1 $0 $a0
sw $t1 0 $zero
addiu $t1 $0 1
sw $t1 4 $zero
jr $ra
addu $t1 $0 $a0
sw $t1 0 $zero
addiu $t1 $0 0
sw $t1 4 $zero
jr $ra
