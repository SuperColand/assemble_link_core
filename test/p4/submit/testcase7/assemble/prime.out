.data
0

.text
00048021
24080001
1208000c
24080002
11100005
0208001b
00008810
12200007
25080001
08000000
00044821
ac090000
24090001
ac090004
03e00008
00044821
ac090000
24090000
ac090004
03e00008

.symbol
0	prime
16	loop
40	is_prime
60	is_not_prime

.relocation
36	loop
